client/blocks/image-editor/index.jsx

```diff
@@ -35,7 +35,7 @@ const ImageEditor = React.createClass( {
 
 	propTypes: {
 		// Component props
-		media: PropTypes.object,
```

Comment:
Good idea. However, I will keep `media` as required prop in README because without it (and `media.URL`, the image editor should/will produce error.